### DISCUSSION TITLE
[错误]

### DISCUSSION THREAD

**Discussion by fasl3** [Chia-Network/chia-blockchain#3251] [General]
Why do you always disconnect at three in the morning？

**Comment by dorkmo** [comment]
some people have an ISP that releases and renews everyone IP every night. someone in germany was saying it was causing them issues everyday.

**Comment by fasl3** [comment]
> some people have an ISP that releases and renews everyone IP every night. someone in germany was saying it was causing them issues everyday.

But my other computer was working just fine, all on the same network

**Reply by keliew** [reply]
A computer working "just fine" is not related to networking issue. A computer can just work without a network.

**Reply by fasl3** [reply]
> A computer working "just fine" is not related to networking issue. A computer can just work without a network.

How do you synchronize without an Internet connection？

**Reply by keliew** [reply]
You could probably synchronise within the LAN...

**Comment by sargonas** [comment]
*auto converting this over to a discussion to continue community support there
